firebase-installations/src/main/java/com/google/firebase/installations/local/PersistedFid.java

```diff
@@ -99,7 +108,7 @@ public synchronized boolean insertOrUpdateCacheEntry(@NonNull FiidCacheEntryValu
   public synchronized boolean clear() {
     SharedPreferences.Editor editor = prefs.edit();
     editor.remove(getSharedPreferencesKey(FIREBASE_INSTALLATION_ID_KEY));
-    editor.remove(getSharedPreferencesKey(CACHE_STATUS_KEY));
+    editor.remove(getSharedPreferencesKey(PERSISTED_STATUS_KEY));
     editor.remove(getSharedPreferencesKey(AUTH_TOKEN_KEY));
```

Comment:
wdyt about this

```
public synchronized boolean clear() {
    SharedPreferences.Editor editor = prefs.edit();
    for (String k : FIID_PREF_KEYS) {
      editor.remove(getSharedPreferencesKey(k));
    }
    editor.commit();
}
```

Reply:
I like it.
Or also
FIID_PREF_KEYS.map(this::getSharedPreferencesKey).each(editor::remove).